db/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM mariadb:10.4.12
+
+COPY ./start.sh /start.sh
+RUN chmod +x /start.sh
```

Comment:
Wait is this actually required? Does docker not COPY with the existing executable permission?